app/src/main/java/com/sun/music_64/genrescreen/LoadContract.java

```diff
@@ -0,0 +1,18 @@
+package com.sun.music_64.genrescreen;
+
+import com.sun.music_64.data.model.Track;
+
+import java.util.List;
+
+public interface LoadContract {
```

Comment:
nên đặt tên cho đồng bộ giữa View và Presenter nữa
```suggestion
public interface GenreContract {
```